projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -6,6 +6,7 @@
  * sharing message.
  */
 
+import { isSimpleSite } from '@automattic/jetpack-shared-extension-utils';
```

Comment:
does `isSimpleSite` also cover Atomic sites?

Reply:
That's a good question Sid, I think not, as there is an `isAtomicSite` function too, but it has to be tested!

Reply:
We tested this together on a pairing session